src/PrestaShopBundle/Controller/Admin/Configure/ShopParameters/MaintenanceController.php

```diff
@@ -0,0 +1,114 @@
+
+use PrestaShopBundle\Controller\Admin\FrameworkBundleAdminController;
+use PrestaShopBundle\Security\Voter\PageVoter;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Tools;
+
+/**
+ * Responsible of "Configure > Shop Parameters > General > Maintenance" page
+ */
+class MaintenanceController extends FrameworkBundleAdminController
+{
+    const CONTROLLER_NAME = 'AdminMaintenance';
+
+    /**
+     * @var FormInterface
+     * @return Response
+     */
+    public function indexAction(Request $request, FormInterface $form = null)
+    {
+        if (is_null($form)) {
+            $form = $this->get('prestashop.adapter.maintenance.form_handler')->getForm();
+        }
+
+        $twigValues = array(
+            'layoutHeaderToolbarBtn' => array(),
+            'layoutTitle' => $this->get('translator')->trans('Maintenance', array(), 'Admin.Navigation.Menu'),
+            'requireAddonsSearch' => true,
+            'requireBulkActions' => false,
+            'showContentHeader' => true,
+            'enableSidebar' => true,
+            'help_link' => $this->generateSidebarLink('AdminMaintenance'),
+            'requireFilterStatus' => false,
+            'form' => $form->createView(),
+            'currentIp' => $request->getClientIp(),
+        );
+
+        return $this->render('@ShopParameters/maintenance.html.twig', $twigValues);
+    }
+
+    /**
+     * @return RedirectResponse
+     */
+    public function processFormAction(Request $request)
+    {
+        $redirectResponse = $this->redirectToRoute('admin_maintenance');
+        if ($this->isDemoModeEnabled()) {
+            $this->addFlash('error', $this->getDemoModeErrorMessage());
+
+            return $redirectResponse;
+        }
+
+        if (!in_array(
+            $this->authorizationLevel($this::CONTROLLER_NAME),
+            array(
+                PageVoter::LEVEL_READ,
+                PageVoter::LEVEL_UPDATE,
+                PageVoter::LEVEL_CREATE,
+                PageVoter::LEVEL_DELETE,
+            )
+        )) {
+            $this->addFlash('error', $this->trans('You do not have permission to update this.', 'Admin.Notifications.Error'));
+            return $redirectResponse;
+        }
+
+        $this->dispatchHook('actionAdminMaintenanceControllerPostProcessBefore', array('controller' => $this));
+        $form = $this->get('prestashop.adapter.maintenance.form_handler')->getForm();
+        $form->handleRequest($request);
+
+        if (!$form->isSubmitted()) {
+            return $redirectResponse;
+        }
+
+        $data = $form->getData();
+        $saveErrors = $this->get('prestashop.adapter.maintenance.form_handler')->save($data);
+
+        if (0 === count($saveErrors)) {
+            $this->addFlash('success', $this->trans('Successful update.', 'Admin.Notifications.Success'));
```

Comment:
Same here, I've got this string in both "/1.7.3/Admin/Notifications/Success.xlf" and "/1.7.3/messages.xlf" (+ "/1.7.2/Admin/Notifications/Success.xlf" but this one is at the right place) folders, should I delete the second one ?

Reply:
I guess we still have occurrences of this string in `messages` in some other place of the project. Removing it may break some translations, isn't it?

Reply:
Is there a way to search for the part(s) of the code where this string appear improperly and modify it with the right domains ?

Reply:
I searched that string in the whole project, but got nothing related to `messages`. It seems we can safely delete it.

Reply:
It is said to be located here: src/PrestaShopBundle/Controller/Admin/AdvancedParameters/PerformanceController.php
I can't check on my own local files right now but it is likely to already have the right domain within the file. It happens sometimes.